operation.go

```diff
@@ -226,7 +226,7 @@ func findInSlice(arr []string, target string) bool {
 }
 
 func (operation *Operation) parseArrayParam(param *spec.Parameter, paramType, refType, objectType string) error {
-	if !IsPrimitiveType(refType) {
+	if !IsPrimitiveType(refType) && !(refType == "file" && paramType == "formData") {
```

Comment:
You are cheating the validation because `file` is not a primitive and I won't allow this hack.

What definition will generate this change if I define type `file` as the following?
```go
type file []string
```

Reply:
<img width="925" alt="Screen Shot 2022-09-19 at 11 50 33 AM" src="https://user-images.githubusercontent.com/996436/190947006-d58f1db6-2454-4dbc-b85a-6e083ae06576.png">

Reply:
I agree this should work as it was provided in the example above but only for`@consume x-www-form-urlencoded` or `@consume application/x-www-form-urlencoded`. 

A mix of `in:formData` with `in:Body` params should not be allowed.